frontend/src/pages/EventCreationPage/components/EventCreationForm/index.tsx

```diff
@@ -29,9 +29,29 @@ interface InitialValuesType {
   hosts: OfficerNameData[];
 }
 
+function findNearestHour(date, isStartDate) {
```

Comment:
Since this is just a helper function, I think it's better to use arrow function syntax here instead. Link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Also, since we are using Typescript (at least for this file), please add type annotations if possible to your function definition (https://www.typescriptlang.org/docs/handbook/2/everyday-types.html)